docs/reference/snapshot-restore/apis/repo-speed-test-api.asciidoc

```diff
@@ -0,0 +1,132 @@
+[role="xpack"]
+[[repo-speed-test-api]]
+=== Repository speed test API
```

Comment:
I originally styled this as an "extended verification API" but feel that verification is too strong a word. We certainly report errors if it finds any, but no errors doesn't mean the repository is 100% correct, nor does it meet any performance threshold. A speed test implies weaker claims so I expect it would be less misleading.

Reverting this back to "extended verification", or calling it something else entirely, is up for discussion.